README.md

```diff
@@ -93,15 +93,17 @@ brew bundle
 
 ### Bootstrap application
 
-1. Clone application as new project with original repository named "rails-base". We use depth parameter here in order not to copy the history of changes in base project
+1. Clone application as new project with original repository named "rails-base" and delete `.git` directory to remove history of changes in base project.
 
    ```bash
-   git clone --depth 1 git://github.com/fs/rails-base.git --origin rails-base [MY-NEW-PROJECT]
+   git clone git://github.com/fs/rails-base.git --origin rails-base [MY-NEW-PROJECT]
+   rm -rf [MY-NEW-PROJECT]/.git
```

Comment:
If you will remove `git` completely we will lose ability to pull changes from upstream (rails-base)